packages/image_picker/image_picker_android/lib/image_picker_android.dart

```diff
@@ -197,6 +202,15 @@ class ImagePickerAndroid extends ImagePickerPlatform {
     return paths.map((dynamic path) => XFile(path as String)).toList();
   }
 
+  @override
+  Future<List<XFile>> getMedia({
+    required MediaOptions options,
+  }) async {
+    final List<String?> paths =
+        await _hostApi.pickMedia(_mediaOptionsToMediaSelectionOptions(options));
+    return paths.map((dynamic path) => XFile(path as String)).toList();
```

Comment:
No `dynamic`, see iOS comments.

If you aren't using method channels directly, you almost never want to be using dynamic. (And even then you don't actually *want* to be, which is why we have Pigeon 😉 )